activerecord/lib/active_record/suppressor.rb

```diff
@@ -30,10 +30,11 @@ module Suppressor
 
     module ClassMethods
       def suppress(&block)
+        previous_state = SuppressorRegistry.suppressed[name] || false
```

Comment:
Yeah I don't have strong feelings on this. Only reason was to make sure `SuppressorRegistry.suppressed[name]` is always a boolean. I'm fine with leaving the value falsy.